src/multi-selector.html

```diff
@@ -2,15 +2,15 @@
 <div class="cp-multi-selector" ng-if="!icon">
 	<input type="input" class="cp-multi-selector__hidden-input" ng-focus="displayDialog()">
 	<div ng-click="displayDialog()" type="text" class="cp-multi-selector__main-input cps-form-control">
-		<div class="cp-multi-selector__pill cps-white cps-bg-gray-10" ng-repeat="item in formatName()">
-			{{item.formatted_name}}<i ng-click="removeItem(item)" class="cps-icon cps-icon-sm-close"></i>
+		<div class="cp-multi-selector__pill cps-white cps-bg-gray-10" ng-repeat="item in selectedItems">
```

Comment:
You may wish to add sorting in here. "item in selectedItems | orderBy: 'display_name'"